src/ReverseProxy/Telemetry/ProxyStage.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.ReverseProxy.Telemetry
+{
+    public enum ProxyStage : int
+    {
+        InvokerSendAsyncStart = 1,
+        InvokerSendAsyncStop,
```

Comment:
There are a few different components named `Invoker`.
```suggestion
        SendAsyncStart = 1,
        SendAsyncStop,
```

Reply:
Renamed